src/Symfony/Component/Security/Core/Tests/Authorization/TraceableAccessDecisionManagerTest.php

```diff
@@ -61,8 +62,8 @@ public static function provideObjectsAndLogs(): \Generator
                 'object' => null,
                 'result' => true,
                 'voterDetails' => [
-                    ['voter' => $voter1, 'attributes' => ['ATTRIBUTE_1'], 'vote' => VoterInterface::ACCESS_GRANTED],
-                    ['voter' => $voter2, 'attributes' => ['ATTRIBUTE_1'], 'vote' => VoterInterface::ACCESS_GRANTED],
+                    ['voter' => $voter1, 'attributes' => ['ATTRIBUTE_1'], 'vote' => new Vote(VoterInterface::ACCESS_GRANTED)],
+                    ['voter' => $voter2, 'attributes' => ['ATTRIBUTE_1'], 'vote' => new Vote(VoterInterface::ACCESS_GRANTED)],
```

Comment:
do we have to change this test case? I mean: this is the sign of a likely BC break, or?